backend/app/infrastructure/routes/projects.py

```diff
@@ -0,0 +1,48 @@
+from flask import current_app as app
+from flask_restx import Namespace, fields
+
+from app.extensions import api
+
+# from app.extensions import cache
+from app.infrastructure import OctantResource
+from app.modules.projects.controller import (
+    get_projects_metadata,
+)
+
+ns = Namespace("projects", description="Octant projects")
+api.add_namespace(ns)
+
+projects_metadata_model = api.model(
+    "ProjectsMetadata",
+    {
+        "projects_addresses": fields.List(
+            fields.String,
+            required=True,
+            description="Projects addresses",
+        ),
+        "projects_cid": fields.String(required=True, description="Projects CID"),
```

Comment:
Please use camelCase as per the general convention for the BE.
```suggestion
        "projectsCid": fields.String(required=True, description="Projects CID"),
```